core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -684,6 +687,7 @@ object KafkaConfig {
       .define(NumIoThreadsProp, INT, Defaults.NumIoThreads, atLeast(1), HIGH, NumIoThreadsDoc)
       .define(BackgroundThreadsProp, INT, Defaults.BackgroundThreads, atLeast(1), HIGH, BackgroundThreadsDoc)
       .define(QueuedMaxRequestsProp, INT, Defaults.QueuedMaxRequests, atLeast(1), HIGH, QueuedMaxRequestsDoc)
+      .define(QueuedMaxBytesProp, LONG, Defaults.QueuedMaxBytes, null, MEDIUM, QueuedMaxBytesDoc)
```

Comment:
Instead of defaulting it to null, should we default it to Defaults.QueuedMaxBytes?